doc/admin-guide/configuration/hrw4u.en.rst

```diff
@@ -0,0 +1,600 @@
+.. Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+   agreements.  See the NOTICE file distributed with this work for additional information regarding
+   copyright ownership.  The ASF licenses this file to you under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with the License.  You may obtain
+   a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software distributed under the License
+   is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+   or implied.  See the License for the specific language governing permissions and limitations
+   under the License.
+
+.. include:: ../../common.defs
+.. default-domain:: cpp
+.. highlight:: none
+
+.. _admin-hrw4u:
+
+HRW4U DSL
+*********
+
+This is a next-generation rule language for the existing ATS ``header_rewrite`` plugin. It
+introduces a clearer syntax, formal grammar, full condition nesting, semantic validation,
+and extensible condition/operator support.
+
+Overview
+========
+
+HRW4U replaces the free-form text parsing of `header_rewrite` with a formally defined
+grammar using ANTLR. This makes HRW4U easier to parse, validate, and extend.
+
+Rather than repeating `header_rewrite` documentation, please refer to:
+  - :ref:`admin-plugins-header-rewrite` for feature behavior and semantics
+  - This page focuses on syntax and behavior *differences* in HRW4U
+
+Why HRW4U?
+----------
```

Comment:
Judging from my observation from the summit, I think we bought the coolaide. I suggest making these _the_ header_rewrite docs. In my opinion we don't need to say "hrw4u" - this is just how you configure header_rewrite now. The old way will be supported through at least 10, and likely 11 too? So we can keep a reference to the old docs and people can use that.